### PR TITLE
Use JSON+LD in Linked Data instead Microdata

### PR DESCRIPTION
**Pull request by ArthurSampaio**
#### What is the purpose of this pull request?

Change the way that Linked Data is injected in source code. Before, this task was using MicroData, now using Json+Ld. 

#### What problem is this solving?

We hope that this changes can fix the bug where the content is inserted in duplicated way.

#### How should this be manually tested?
WS: https://mdsku--storecomponents.myvtex.com/shirt/p
Go to the  [google test tool](https://search.google.com/structured-data/testing-tool) and paste the source code from the page of a product.

#### Types of changes

* [ ] Bug fix (a non-breaking change which fixes an issue)
* [x] New feature (a non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/StructuredData.js [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+    "@context": "https://schema.org/",
+    "@type" : "Product",
+    "name": "${name}", 
+    "brand": "${brand}",
+    "image": "${image.imageUrl}",
+    "description": "${description}",
+    "mpn": "${product.productId}",
+    "sku": "${skuId}",
+    "offers": ${composeAggregateOffer(product, currency)}
+
```
Comment (luciannojunior) [inline react/components/StructuredData.js]: Run prettier

File: react/components/StructuredData.js [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+    "mpn": "${product.productId}",
+    "sku": "${skuId}",
+    "offers": ${composeAggregateOffer(product, currency)}
+    
+  }`
+
+  return productLD
+}
+
+export default function StructuredData({ product, query }, { culture: { currency, locale } }) {
```
Comment (luciannojunior) [inline react/components/StructuredData.js]: Export the component at the bottom of the page, it improves readability

Comment (luciannojunior) [reply react/components/StructuredData.js]: And also, get that properties that you are getting with the [legacy API](https://reactjs.org/docs/legacy-context.html) with `useRuntime`. Refer [here](https://github.com/vtex-apps/store-header/blob/5b995ce600cf4ed06c2e4d837f196750ddf95c08/react/hooks/useDevice.js#L10) for usage

File: react/ProductWrapper.js [hunk 3]
```diff
@@ -117,12 +118,9 @@ class ProductWrapper extends Component {
       <div className="vtex-product-context-provider">
         <Helmet>
           {titleTag && <title>{titleTag}</title>}
-          {metaTagDescription && (
```
Comment (luciannojunior) [inline react/ProductWrapper.js]: Wouldn't be better for you to include the script tag in `Helmet`?

Comment (ArthurSampaio) [reply react/ProductWrapper.js]: When i put in helmet scope the google tool doesnt regonize the metadata.

File: react/components/StructuredData.js [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+    />
+  );
+}
+
+StructuredData.propTypes = {
+  product: PropTypes.object,
+  query: PropTypes.object
+};
+
+StructuredData.contextTypes = {
```
Comment (luciannojunior) [inline react/components/StructuredData.js]: Remove it since it's no longer needed

**Comment by luciannojunior** [comment]
Very cool feature!

I believe that you would be a great fit for looking into more ways we can improve our metadata. I've seen some of the Americana's and there are some features we are able to implement as of now, and other we can get that data from our API int the future (like reviews) 😄

File: react/ProductWrapper.js [hunk 5]
```diff
@@ -1,10 +1,11 @@
-import PropTypes from 'prop-types'
-import React, { Component, Fragment } from 'react'
-import { last, head } from 'ramda'
-import { Helmet, withRuntimeContext } from 'vtex.render-runtime'
+import PropTypes from "prop-types";
```
Comment (brunojdo) [inline react/ProductWrapper.js]: wat?

Comment (ArthurSampaio) [reply react/ProductWrapper.js]: prettier.

Comment (ArthurSampaio) [reply react/ProductWrapper.js]: i already remove the semicolons

**Pull request merged**